sklearn/metrics/tests/test_score_objects.py

```diff
@@ -47,7 +51,7 @@ class EstimatorWithoutFit(object):
     pass
 
 
-class EstimatorWithFit(object):
+class EstimatorWithFit(BaseEstimator):
```

Comment:
No. Clone has a ``safe=False`` option that does that, but we are not using that in ``cross_val_score`` or anywhere else I know of.